awscli/autocomplete/local/basic.py

```diff
@@ -496,3 +496,78 @@ def _get_struct_keys_completions(self, arg_model, entered_keys,
                     )
                 )
         return self._filter(last_key, results)
+
+
+class QueryCompleter(BaseCompleter):
+
+    def __init__(self, cli_driver_fetcher=None,
+                 response_filter=startswith_filter):
+        self._filter = response_filter
+        self._cli_driver_fetcher = cli_driver_fetcher
```

Comment:
One thing that I ran into was that I tried this with the tab completer (e.g. `aws_completer). Specifically, I tried:
```
aws ec2 describe-regions --query <TAB>
```
And it printed out the following Traceback:
```
Traceback (most recent call last):
  File "/Users/kyleknap/.pyenv/versions/pyinstaller-3.7.2/bin/aws_completer", line 7, in <module>
    exec(compile(f.read(), __file__, 'exec'))
  File "/Users/kyleknap/GitHub/aws-cli/bin/aws_completer", line 36, in <module>
    main()
  File "/Users/kyleknap/GitHub/aws-cli/bin/aws_completer", line 28, in main
    autocomplete(command_line, command_index)
  File "/Users/kyleknap/GitHub/aws-cli/awscli/autocomplete/main.py", line 55, in autocomplete
    results = completer.autocomplete(command_line, position)
  File "/Users/kyleknap/GitHub/aws-cli/awscli/autocomplete/completer.py", line 44, in autocomplete
    result = completer.complete(parsed)
  File "/Users/kyleknap/GitHub/aws-cli/awscli/autocomplete/local/basic.py", line 527, in complete
    operation_model = self._cli_driver_fetcher.get_operation_model(
AttributeError: 'NoneType' object has no attribute 'get_operation_model'
```
I think at a minimum it should be not allow an error to propagate like this.